Drawer: update revealMode description (#4904)
DevExpress/devextreme-documentation: api-reference/10 UI Components/dxDrawer/1 Configuration/revealMode.md

* *"expand"*    
The drawer expands from the closed position. The drawer's width is animated; its content is not.

> When you change Drawer's reveal mode, the component needs to perform recalculations for animation. As a result, the [template](/Documentation/ApiReference/UI_Components/dxDrawer/Configuration/#template) is rerendered which may produce side effects like additional data loading.

Reviewer: ```suggestion
When you change the Drawer's reveal mode, the component needs to perform recalculations for animation. As a result, the [template](/Documentation/ApiReference/UI_Components/dxDrawer/Configuration/#template) is rerendered which can increase loading time.
```
